src/components/live/Landing.jsx

```diff
@@ -1,5 +1,16 @@
 const Landing = () => {
-  return <div className="flex items-center justify-center">Landing</div>;
+  return (
+    <div className="flex justify-evenly bg-[#021334] p-6 font-semibold text-white">
+      <div className="p-2">HOME</div>
+      <div className="p-2">SCHEDULE</div>
+
```

Comment:
make a navigation.js file in data folder 
you will store the data there 
i.e 

```
export const items = [
{
name: "HOME"
link:"/"
},
{
name:"ABOUT"
link:"#about"
},
] 
```
etc